packages/client-node/src/connection/node_base_connection.ts

```diff
@@ -358,52 +161,6 @@ export abstract class NodeBaseConnection
     }
   }
 
-  async exec(
-    params: ConnBaseQueryParams,
-  ): Promise<ConnExecResult<Stream.Readable>> {
-    const query_id = this.getQueryId(params.query_id)
-    const searchParams = toSearchParams({
-      database: this.params.database,
-      clickhouse_settings: params.clickhouse_settings,
-      query_params: params.query_params,
-      session_id: params.session_id,
-      query_id,
-    })
-    const { controller, controllerCleanup } = this.getAbortController(params)
-    try {
-      const { stream, summary } = await this.request(
-        {
-          method: 'POST',
-          url: transformUrl({ url: this.params.url, searchParams }),
-          body: params.query,
-          abort_signal: controller.signal,
-          parse_summary: true,
-        },
-        'Exec',
-      )
-      return {
-        stream,
-        query_id,
-        summary,
-      }
-    } catch (err) {
-      controller.abort('Exec HTTP request failed')
-      this.logRequestError({
-        op: 'Exec',
-        query_id: query_id,
-        query_params: params,
-        search_params: searchParams,
-        err: err as Error,
-        extra_args: {
-          clickhouse_settings: params.clickhouse_settings ?? {},
-        },
-      })
-      throw err // should be propagated to the user
-    } finally {
-      controllerCleanup()
-    }
-  }
```

Comment:
This one was used for both exec + command client methods. Now, the connection has them separate, but the logic is still (mostly) shared.